CONTRIBUTING.rst

```diff
@@ -59,7 +59,7 @@ Coding Conventions
 * Functions should do one thing.
 * Early pull requests and code reviews.
 * Early architecting/design. Code reviews can happen before any code has been written.
-* Use a consistent character width of 120.
+* Use a consistent character width limit of 120.
```

Comment:
I think people will see this and will use 120 as their standard length. We should say here our recommended 100 (with an asterisk) and let the  pep-8 section then present the hard limit of 120